### ISSUE TITLE
Mysql timezone

### ISSUE THREAD

**Issue by Arcanum417** [alexcroox/R3-Web#11]
On the Mysql server I migrated to I cannot change the timezone. Timezone could be setup session by session or globally. It would be nice to set timezone for the addon part and timezone for the php part.

**Comment by alexcroox** [comment]
The addon part uses `NOW()` for its date time saving which means it will use whatever the MySQL db timezone is set to. To get the php part to match that, set the timezone in `config.php`

**Comment by Arcanum417** [comment]
Yeah, I got that. However I cannot change Mysql server timezone but I want to show [Here](http://i.imgur.com/0oH1Dlb.png) diffrent timezone as is saved on Mysql server

**Comment by Arcanum417** [comment]
It looks like it would be easiest to manipulate the value while rendering the table. So backend still works with with Mysql timezone but you can set in config timezone for the frontend.
